app/redux/wallet/actions.ts

```diff
@@ -170,15 +198,19 @@ export const restoreFile = ({ filePath }: { filePath: string }) => async (dispat
 };
 
 export const backupWallet = () => async (dispatch: AppThDispatch, getState: GetState) => {
-  const { walletFiles } = getState().wallet;
-  const { error } = await eventsService.copyFile({ filePath: walletFiles ? walletFiles[0] : '', copyToDocuments: true });
+  const { currentWalletPath } = getState().wallet;
+  if (!currentWalletPath) {
+    dispatch(setUiError(new Error('Can not create wallet backup: Wallet does not opened')));
+    return null;
+  }
+  const { error, filePath } = await eventsService.backupWallet(currentWalletPath);
   if (error) {
     console.log(error); // eslint-disable-line no-console
     dispatch(setUiError(addErrorPrefix('Can not create wallet backup\n', error)));
-    return false;
+    return null;
```

Comment:
is it possible throw an Error in case of error?

Reply:
Then we still have to handle it somewhere.
But probably we'd like to handle such cases within #882. So I propose to solve this issue within this task.
If you want, we can discuss the solution :)

Reply:
yes, lets resolve it in 882